pkg/steps/project_image.go

```diff
@@ -50,10 +50,12 @@ func (s *projectDirectoryImageBuildStep) Run(ctx context.Context, dry bool) erro
 
 	labels := make(map[string]string)
 	if len(s.jobSpec.Refs.Pulls) == 0 {
-		labels["io.openshift.build.commit.id"] = s.jobSpec.Refs.BaseSHA
+		labels["org.opencontainers.image.source"] = fmt.Sprintf("https://github.com/%s/%s", s.jobSpec.Refs.Org, s.jobSpec.Refs.Repo)
```

Comment:
Should these end in .git?

We need the ref for other reasons.  We probably also need to correctly indicate merge commits.

Reply:
> Should these end in .git?

You can `pull` either way.  And hitting a`.git` in my browser redirects me to the non-`.git` form, so *shrug*.  I like the way I have it, but will change if you like.

> We need the ref for other reasons.

What ref, the old `io.openshift.build.commit.id`?

> We probably also need to correctly indicate merge commits.

Follow-up work?